api/authenticator.py

```diff
@@ -921,12 +921,31 @@ def create_authentication_document(self):
             links=links
         ).to_dict(self._db)
 
-        # Add the library's color scheme, if it has one.
+        # Add the library's mobile color scheme, if it has one.
         description = ConfigurationSetting.for_library(
             Configuration.COLOR_SCHEME, library).value
         if description:
             doc['color_scheme'] = description
 
+        # Add the library's web colors, if it has any.
+        background = ConfigurationSetting.for_library(
+            Configuration.WEB_BACKGROUND_COLOR, library).value
+        foreground = ConfigurationSetting.for_library(
+            Configuration.WEB_FOREGROUND_COLOR, library).value
+        if background or foreground:
+            doc["web_colors"] = dict(background=background, foreground=foreground)
```

Comment:
I would still like to see this called `web_color_scheme` to match `color_scheme`.